bodhi/tests/server/tasks/test_composer.py

```diff
@@ -2344,14 +2324,14 @@ def test_request_not_stable(self, Popen):
                        'testcontainer2:1.0.1-1.fc28container'):
             for dtag in [source.split(':')[1], source.split(':')[1].split('-')[0], 'testing']:
                 mock_call = mock.call(
-                    [config['skopeo.cmd'], 'copy',
+                    [config['skopeo.cmd'], 'copy', '--dest-tls-verify=false',
```

Comment:
This looks like it's mistakenly copied from `test_skopeo_extra_copy_flags()`. It breaks this and a couple of other tests for me, at any rate. :wink:

Reply:
I will remove it than.